apps/discord-bot/src/structures/commandOptions/allUserPermissions.ts

```
@@ -0,0 +1,34 @@
+          .setTimestamp()
+          .setAuthor({
+            name: message.member?.user.globalName ?? message.member?.user.username ?? '',
+            iconURL: message.member?.user.displayAvatarURL(),
+          })
+          .setThumbnail(client.user.displayAvatarURL())
+          .setDescription(`You are missing the set permissions which are necessary to run this command. Please acquire these permissions to execute this command:\n${missingPermissions.map((permission: string) => `↳ \`${permission}\``).join('\n')}`),
+      ],
+    })
+    return false
+  };
+}
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement, ...

... this statement, ...

... this statement, ...

... this statement, ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)